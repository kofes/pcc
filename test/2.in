12345.67890 'abcde' 'a'
1234567890
[] ( ) ,
: ... ;
+ := / = > >= < <= <>
* -
^ @
#52 #53 5
{
  somewords
}
//some words
(* some
        words*)
