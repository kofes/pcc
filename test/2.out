1:1	LITERAL	106	12345.67890
1:13	LITERAL	107	abcde
1:21	LITERAL	108	a
2:1	LITERAL	86	1234567890
3:1	PUNCTUATION	112	[
3:2	PUNCTUATION	113	]
3:4	PUNCTUATION	116	(
3:6	PUNCTUATION	117	)
3:8	PUNCTUATION	118	,
4:1	PUNCTUATION	119	:
4:3	PUNCTUATION	121	..
4:5	PUNCTUATION	120	.
4:7	PUNCTUATION	122	;
5:1	OPERATOR	123	+
5:3	OPERATOR	125	:=
5:6	OPERATOR	127	/
5:8	OPERATOR	128	=
5:10	OPERATOR	129	>
5:12	OPERATOR	130	>=
5:15	OPERATOR	132	<
5:17	OPERATOR	133	<=
5:20	OPERATOR	135	<>
6:1	OPERATOR	136	*
6:3	OPERATOR	141	-
7:1	OPERATOR	143	^
7:3	OPERATOR	144	@
8:1	LITERAL	108	4
8:5	LITERAL	108	5
8:9	LITERAL	86	5
15:1	END_OF_FILE	0	
